developer-resources/tools/IntelliJ.md

```diff
@@ -19,11 +19,26 @@ https://www.jetbrains.com/idea/download/index.html
 There is a [tutorial for IntelliJ](../../open-metadata-resources/open-metadata-tutorials/intellij-tutorial) as part of the
 [Egeria Dojo](../../open-metadata-resources/open-metadata-tutorials/egeria-dojo).
 
-Lombok Plugin
+## Lombok Plugin
 
 Egeria makes use of Project Lombok. If using JetBrains IntelliJ IDEA ensure the IDEA has the required plugin configured. See https://projectlombok.org/setup/intellij for more information.
 Also before running a Maven build please choose "Don't detect" from the "Generated sources folders" dropdown in Preferences -> Build, Execution, Deployment -> Build Tools -> Maven -> Importing. This will avoid triggering a duplicate classes build error caused by delomboked sources folder being added as source folder for the Maven module.
 
+If this wasn’t set when your project was initially setup, you may find that ‘delombok’ directories are already present in IntelliJ’s source path for some modules, leading to errors with duplicate classes.
+
+To check for any modules still refering to delombok you can run this at the command line, from your top level source tree:
+
+```
+find . -name '*.iml' | xargs -n50 grep -y delombok
+```
+If you find any hits such as:
+```
+./open-metadata-implementation/access-services/data-engine/data-engine-api/data-engine-api.iml:      <sourceFolder url="file://$MODULE_DIR$/target/delombok" isTestSource="false" />
+```
+
+then either remove those lines WITH INTELLIJ NOT RUNNING, or go into File->Project Structure->Modules, and remove ’target/delombok' from the ‘Source Folders’ list
+
+Explanation - In addition to importing importation on source files from maven, IntelliJ also tries to look for generated source. It finds the delombok directory causing duplicates — in fact we only use this directory for generating javadoc of lombok-enabled modules. Switching the setting/removing these source folders prevents duplicate classes.
```

Comment:
I think importing importation sounds a bit redundant to me. Otherwise, all looks very detailed and useful.

Reply:
I have the same remark but I also feel like I am missinterpreting the phrase since I am not native English spearker :)

Reply:
Maybe

Explanation - in addition to importing project definitions from maven pom.xml, IntelliJ also .....

Reply:
I'll update...

Reply:
@lpalashevski @alexandra-bucur Updated